components/brave_ads/core/internal/account/utility/redeem_confirmation/reward/redeem_reward_confirmation.cc

```diff
@@ -105,7 +108,20 @@ void RedeemRewardConfirmation::CreateConfirmationCallback(
   BLOG(6, UrlResponseToString(url_response));
   BLOG(7, UrlResponseHeadersToString(url_response));
 
-  FetchPaymentToken(std::move(redeem_confirmation), confirmation);
+  FetchPaymentTokenAfter(kFetchPaymentTokenAfter.Get(),
```

Comment:
I do not see any changes to `redeem_reward_confirmation_unittest` to cover `FetchPaymentTokenAfter`?

Reply:
We have the test: `BraveAdsRedeemRewardConfirmationTest.FetchPaymentTokenAfter`

Reply:
Great, thanks